src/EventStore.Core/ClusterVNode.cs

```diff
@@ -777,6 +800,8 @@ public class ClusterVNode :
 		}
 
 		public void Handle(SystemMessage.BecomeShuttingDown message) {
+			UnixSignalManager.StopProcessing();
```

Comment:
We should be careful to ensure that this doesn't break SIGINT while shutting down, which should immediately terminate the process.

Reply:
@jen20 yup, thanks, I've verified that it doesn't break it and does a proper shut down. We're currently handling only signals that we subscribe to in the UnixSignalManager (only SIGHUP currently), any other signal will still use its default handler.